src/intl/en/common.json

```diff
@@ -62,12 +62,15 @@
   "ethereum-2-0": "Ethereum 2.0",
   "ethereum-brand-assets": "Ethereum Brand Assets",
   "ethereum-community": "Ethereum Community",
+  "ethereum-online": "Online communities",
```

Comment:
More of a general note, but we have some mixed casing in the Community drop down now with these. There is a bit of this throughout all of the drop downs, but we should probably look at addressing that.